mediainfo.yaml

```diff
@@ -0,0 +1,63 @@
+package:
+  name: mediainfo
+  version: 25.03
+  epoch: 0
+  description: "Utility for displaying technical and tag data for media files"
+  copyright:
+    - license: BSD-3-Clause
+
+environment:
+  contents:
+    packages:
+      - busybox
+      - ca-certificates
+      - git
+      - build-base
+      - autoconf
+      - automake
+      - libtool
+      - pkgconf
+      - pkgconf-dev
+      - glibc
```

Comment:
I wouldn't explicitly build-dep on glibc. The things that need it will already depend on it.